packages/ember-routing-htmlbars/tests/helpers/closure_action_test.js

```diff
@@ -268,6 +267,22 @@ QUnit.test('an error is triggered when bound action function is undefined', func
   }, /An action could not be made for `somethingThatIsUndefined` in .*\. Please confirm that you are using either a quoted action name \(i\.e\. `\(action 'somethingThatIsUndefined'\)`\) or a function available in .*\./);
 });
 
+QUnit.test('[#12718] a nice error is shown when a bound action function is undefined and it is passed as attrs.foo', function(assert) {
+  registerComponent('inner-component', EmberComponent.extend({
+    [OWNER]: owner
+  }));
+  registerTemplate('components/inner-component', '<button id="inner-button" {{action (action attrs.external-action)}}>Click me</button>');
+
+  view = EmberComponent.extend({
+    layout: compile('{{inner-component}}'),
+    [OWNER]: owner
+  }).create();
+
+  throws(function() {
+    runAppend(view);
+  }, /Action passed is null or undefined in \(action [^)]*\)./);
```

Comment:
ya, that would be sweet. But this already is a big win, I was just thinking what would make it even better.